samples: matter: Pulled update removing defaults for child images (#15575)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* Updated default MRP retry intervals for Thread devices to two seconds to reduce the number of spurious retransmissions in Thread networks.
* Increased the number of available packet buffers in the Matter stack to avoid packet allocation issues.

> * Removed Kconfig :file:`Kconfig.mcuboot.defaults`, :file:`Kconfig.hci_ipc.defaults` and :file:`Kconfig.multiprotocol_rpmsg.defaults` files storing a default configuration for the child images.

Reviewer: ```suggestion
* Removed the :file:`Kconfig.mcuboot.defaults`, :file:`Kconfig.hci_ipc.defaults` and :file:`Kconfig.multiprotocol_rpmsg.defaults` Kconfig files that stored a default configuration for the child images.
```

>   This was done because of the sysbuild intergration and the child images reprecation.

Reviewer: ```suggestion
  This was done because of the sysbuild integration and the child images deprecation.
```

>   The configuration should be applied using the configuration files located in the sample's or applications's directory.

Reviewer: ```suggestion
  The configurations are now applied using the configuration files located in the sample's or application's directory.
```

>   To see how to migrate an application from the previous to the current approach see the :ref:`migration guide <migration_2.7>`.

Reviewer: ```suggestion
  To see how to migrate an application from the previous to the current approach, see the :ref:`migration guide <migration_2.7>`.
```
Should also add an empty line before this, so it gets its own paragraph in the bullet.
